package-lock.json

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
Why do you have this file when you don't have a `package.json`?

Reply:
I have no idea. I removed it. The app runs fine, it doesn't seem to actually do anything.